### PR TITLE
Small English pass

### PR DESCRIPTION
**Pull request by IlDucci**
- Changed Bennet's romanization to Bennett, following the official game's translation.
 - Found quite a lot of missing characters that were replaced by periods. So far I've only covered Cybil's scenario.
 - Changed the GBA button names to follow the official naming conventions.
 - Other fixes here and there, small for now.

### CODE REVIEW

File: scripts/EN/Cybil [hunk 1]
```diff
@@ -1311,7 +1311,7 @@ ShowText((7052)"The bullets scattered about on the floor supported this, as they
 HandleInput(); // 0x1a02: 00000010
 ShowText((7053)"Other than this there did not appear to be anything else out of the ordinary."); // 0x1a06: 0000100c
 HandleInput(); // 0x1a0c: 00000010
-ShowText((7054)"¥"Well, your Daddy is not here. Let's look some place else.¥" Cybil took Cheryl's hand, and without a word they rushed out into the snowy street once again."); // 0x1a10: 0000100c
```
Comment (07151129) [inline scripts/EN/Cybil:1314]: This looks fine to me. [NOAD](https://en.wikipedia.org/wiki/New_Oxford_American_Dictionary) says:

> some·place | ˈsəmˌplās | North American informal
> adverb
> in or to some unspecified place; somewhere: let's go someplace private to talk.
> pronoun
> some unspecified place; somewhere: we drove around, looking for someplace to park.

Comment (IlDucci) [reply scripts/EN/Cybil:1314]: Restored.

**Comment by IlDucci** [comment]
Sorry, I missed the other duplicate of "some place else"!

**Comment by 07151129** [comment]
Thanks for proofreading. The missing letters were caused by weird formatting in the original translation .txt which my script failed to process properly.

I'm not sure why Toby misspelt Cybil's name given the fact that he consulted the official PSX translation.

**Pull request merged**